### PR TITLE
 Launch grid template implementation.

### PR DESCRIPTION
**Pull request by ArtsiomYeliseyenka**

### CODE REVIEW

File: app/src/common/utils/index.js [hunk 1]
```diff
@@ -27,4 +27,5 @@ export { referenceDictionary } from './referenceDictionary';
 export { fetch, ERROR_CANCELED, ERROR_UNAUTHORIZED } from './fetch';
 export { addTokenToImagePath } from './addTokenToImagePath';
 export { getStorageItem, setStorageItem } from './storageUtils';
+export { getDuration, approximateTimeFormat, dateFormat, fromNowFormat, daysFromNow, daysBetween } from './timeDateUtils';
```
Comment (SergeyKovalchuk) [inline app/src/common/utils/index.js]: Please use style that is common in such situations like 
```javascript
{
   item,
   item1
}
```

File: app/src/components/inputs/inputCheckbox/inputCheckbox.jsx [hunk 2]
```diff
@@ -49,7 +49,7 @@ export const InputCheckbox = ({ children, value, disabled, onChange, onFocus, on
           <polygon fill={disabled ? '#999' : '#fff'} fillRule="evenodd" points="0 3.111 3 6.222 8 1.037 7 0 3 4.148 1 2.074" />
         </svg>
       </div>
-      <span className={cx({ 'children-container': true, disabled })}>{children}</span>
+      { children ? <span className={cx({ 'children-container': true, disabled })}>{children}</span> : null }
```
Comment (SergeyKovalchuk) [inline app/src/components/inputs/inputCheckbox/inputCheckbox.jsx]: We are using React of 16 version, please use an appropriate syntax for conditional rendering 
```javascript 
condidtion && render()
```

File: app/src/components/main/grid/grid.jsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+const cx = classNames.bind(styles);
+
+export const Grid = ({ columns, data }) => (
+  <div className={cx('grid')}>
+    <GridHeader columns={columns} />
+    <GridBody columns={columns} data={data} />
+  </div>
+);
+Grid.propTypes = {
```
Comment (SergeyKovalchuk) [inline app/src/components/main/grid/grid.jsx]: Why are using class registration of propTypes and defaultProps  in AbsRelTime. 
We should use common approach

File: app/src/pages/inside/launchesPage/launcheSuiteGrid/itemInfo/durationBlock/durationBlock.jsx [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+import { injectIntl, intlShape, defineMessages } from 'react-intl';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import ClockIcon from './img/clock-icon-inline.svg';
+import InProgressGif from './img/item-in-progress.gif';
+
+import styles from './durationBlock.scss';
+
+const cx = classNames.bind(styles);
+const messages = defineMessages({
```
Comment (SergeyKovalchuk) [inline app/src/pages/inside/launchesPage/launcheSuiteGrid/itemInfo/durationBlock/durationBlock.jsx]: such messages should be defined in high level

File: app/src/components/inputs/inputCheckbox/inputCheckbox.jsx (hunk 2 above)
Comment (ArtsiomYeliseyenka) [reply app/src/components/inputs/inputCheckbox/inputCheckbox.jsx]: fixed

File: app/src/components/main/grid/grid.jsx (hunk 3 above)
Comment (ArtsiomYeliseyenka) [reply app/src/components/main/grid/grid.jsx]: TBD

File: app/src/pages/inside/launchesPage/launcheSuiteGrid/itemInfo/durationBlock/durationBlock.jsx (hunk 4 above)
Comment (ArtsiomYeliseyenka) [reply app/src/pages/inside/launchesPage/launcheSuiteGrid/itemInfo/durationBlock/durationBlock.jsx]: TBD

**Pull request closed without merging**